v19.2/explain.md

```diff
@@ -18,6 +18,8 @@ Using `EXPLAIN`'s output, you can optimize your queries by taking the following
 
 - Avoid scanning an entire table, which is the slowest way to access data. You can avoid this by [creating indexes](indexes.html) that contain at least one of the columns that the query is filtering in its `WHERE` clause.
 
+- <span class="version-tag">New in v19.2:</span> By default, `vectorize=auto`, meaning the column-oriented vectorized SQL execution engine is enabled for all streaming operations. If you are querying a table with a small number of rows, it might be more efficient to use row-oriented execution. The `vectorize_row_count_threshold` [cluster setting](cluster-settings.html) specifies the minimum number of rows required to use the vectorized engine to execute a query plan.
```

Comment:
Same comment as above re: I don't think "streaming operations" is something we can expect most users to understand without explanation.  It would be more understandable IMO to avoid saying "supports streaming operations" and instead say something like "the vectorized engine is enabled for many SQL queries that would benefit from columnarized execution.  For details about queries supported by the vectorized engine, see '[Vectorized Execution]()'" (or some such).  And then list out all the streaming vs. non-streaming and which queries are what on the separate '[Vectorized Execution]()' page.